src/strategies/aavegotchi-agip-17/index.ts

```diff
@@ -0,0 +1,66 @@
+import { subgraphRequest } from '../../utils';
+
+export const author = 'candoizo';
+export const version = '0.1.0';
+
+const AAVEGOTCHI_SUBGRAPH_URL = {
+  137: 'https://api.thegraph.com/subgraphs/name/aavegotchi/aavegotchi-core-matic'
+};
+
+export async function strategy(
+  _space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
```

Comment:
```
'snapshot' is defined but never used
```
Hi @candoizo is there some way to pass snapshot block number into the subgraph? without it, the voting power may change in the future or in the middle of voting a proposal etc. the VP should be based on snapshot block number to have consistent results

Reply:
good point, fixed in recent commit thanks